draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -562,11 +578,11 @@ The Internet provides no guarantee that different paths
 (e.g., using different endpoint addresses) are disjoint.
 This has additional implications:
 New CCs MUST evaluate the potential
-harm to other flows when the multiple paths share a common
+harm toimpact on other flows when the multiple paths share a common
 congested bottleneck
```

Comment:
harm to, impact on, harm to or impact on? But surely not "harm toimpact on other flows".